mautic/mautic
# Fix report owners filter

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Noticed in reports options missing for owner filter.




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contact report
3. Try Owners filter. See cannot choose any owner
4. After PR it works properly
